cmd/http-server/main.go

```diff
@@ -7,12 +7,14 @@ SPDX-License-Identifier: Apache-2.0
 package main
 
 import (
-	"log"
-
 	"github.com/spf13/cobra"
+	"github.com/trustbloc/edge-core/pkg/log"
```

Comment:
"github.com/spf13/cobra"

	"github.com/trustbloc/edge-core/pkg/log"
	"github.com/trustbloc/edge-agent/cmd/http-server/startcmd"

Reply:
Same response as my previous comments - so far we've been grouping other Trustbloc projects along with third party libraries.